test/overlay.html

```diff
@@ -347,6 +360,35 @@
         });
       });
     });
+
+    describe('frontmost', () => {
+      var dialog;
+      beforeEach(() => {
+        dialog = fixture('front-most');
+      });
+
+      it('should bring to frontmost on mousedown', (done) => {
+        dialog.opened = true;
+        const dialogOverlay = dialog.$.overlay;
+        const datePicker = dialog.$.overlay.querySelector('vaadin-date-picker');
+    
+        const datePickerPosition = datePicker.getBoundingClientRect();
```

Comment:
Rather use the position of the date picker _overlay_

Reply:
done